client/src/main/java/io/pravega/client/stream/impl/CheckpointState.java

```diff
@@ -196,6 +196,23 @@ List<String> getOutstandingCheckpoints() {
                           .collect(Collectors.toList());
     }
 
+    /**
+     * Removes the outstanding checkpoints.
+     * @return false when there are no outstanding checkpoints otherwise true
+     */
+    boolean removeOutstandingCheckpoints() {
+       List<String> checkpoint = getOutstandingCheckpoints();
+       if (checkpoint.isEmpty()) {
```

Comment:
If checkpoint map is empty then log message and return true

Reply:
If we are always returning true, in that case its redundant to put return type Boolean. May be we can make it void and put some logs in the method ?